doc/metadata.txt

```diff
@@ -1588,15 +1588,15 @@ The default is `false`, which means that all devices can be hotplugged.
 ```{config:option} gid device-unix-usb-device-conf
 :condition: "container"
 :defaultdesc: "`0`"
-:shortdesc: "GID of the device owner in the container"
+:shortdesc: "GID of the device owner in the instance"
```

Comment:
Two lines before the changed line says:
```:condition: "container"``` 
(line 1589)

Does this mean that this option only exists on containers? Can you confirm whether this device option is also available to instances, and if so, what the `condition` property refers to? If this is an option that is only available when using a container, then the shortdescs should stay as-is.

Same for the other two similar changes on lines 1599 and 1626.

Reply:
The `usb` device type is available for both containers and VMs. The idea here is to consistently use the word `instance` for `usb` device options, but keep the `container` condition since the `uid`, `gid`, and `mode` options only apply to containers.